src/drive/web/modules/drive/Toolbar/components/AddEncryptedFolderItem.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import { ActionMenuItem } from 'cozy-ui/transpiled/react/ActionMenu'
+import EncryptedFolderIcon from 'drive/web/modules/views/Folder/EncryptedFolderIcon'
+import { useVaultUnlockContext } from 'cozy-keys-lib'
+import {
+  showNewFolderInput,
+  encryptedFolder
+} from 'drive/web/modules/filelist/duck'
+
+const AddEncryptedFolderItem = translate()(({ t, addEncryptedFolder }) => {
```

Comment:
you can remove the `translate()` HOC and use the `use18n()` hook inside your component.